api_tests/lib_sdk/actors/actor.ts

```diff
@@ -402,18 +406,17 @@ export class Actor {
                 );
             }
         }
-    }
```

Comment:
Pretty sure this bracket is needed for this to be valid TS.

Reply:
Reading from bottom of this page to the top, this is what caused the reformat below.